.github/workflows/cd.yaml

```diff
@@ -0,0 +1,32 @@
+name: Update Jenkins plugins
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+     - 'master'
```

Comment:
doesn't really need to run on push, but this will trigger it every time Jenkins is updated which is probably a good thing in case any plugins require that specific version